packages/beacon-node/src/network/reqresp/handlers/dataColumnSidecarsByRange.ts

```diff
@@ -26,10 +26,14 @@ export async function* onDataColumnSidecarsByRange(
 
       for (const [index, dataColumnSidecarBytes] of dataColumnSidecars.entries()) {
         if (!dataColumnSidecarBytes) {
-          throw new ResponseError(
-            RespStatus.SERVER_ERROR,
-            `No finalized dataColumnSidecar found for slot=${slot}, index=${columns[index]}`
-          );
+          chain.logger.debug("No finalized dataColumnSidecar found.", {
+            slot,
+            index: columns[index],
+            requestStartSlot: startSlot,
+            requestCount: count,
+            requestColumns: columns.join(","),
+          });
+          continue;
```

Comment:
Logging seem ok here but I think if we do that we should check the requested columns against the custodyConfig and remove indices that we know we do not custody.  Requests should likely already be pruning stuff the peer does not think we have but would be good to tell the difference between columns we should have and do not from columns we should not have.